src/execution_plan/ops/op.h

```diff
@@ -49,7 +49,7 @@ typedef enum {
 	OPType_SEMI_APPLY = (1 << 29),
 	OpType_ANTI_SEMI_APPLY = (1 << 30),
 	OPType_OR_APPLY_MULTIPLEXER = (1 << 31),
-	OPType_AND_APPLY_MULTIPLEXER = (1 << 31),
+	OPType_AND_APPLY_MULTIPLEXER = (1 << 31), // TODO same value as OR_APPLY
```

Comment:
TODO or change?

Reply:
TODO for now - there's no code that relies on differentiating between OR and AND multiplexer types, and the fix here will probably force us to give up on using this enum as a bitmask.

Reply:
I think we need to change that
`OPType_AND_APPLY_MULTIPLEXER = (1 << 32)`
this might bite us at some point.